openshift-hack/conformance-k8s.sh

```diff
@@ -15,80 +15,87 @@ fi
+   to run the conformance tests. You will need to add the `--dns-namespace=openshift-dns`
+   and `--dns-pod-labels=dns.operator.openshift.io/daemonset-dns=default`
+   options so `sonobuoy` can find the cluster DNS pods:
 
-    export KUBECONFIG=PATH_TO_KUBECONFIG
+```
+sonobuoy run --mode=certified-conformance --dns-namespace=openshift-dns --dns-pod-labels=dns.operator.openshift.io/daemonset-dns=default
+```
 
-3. Clone the OpenShift source repository and change to that directory:
+3. Once conformance testing is completed, restore the default security rules:
 
-    git clone https://github.com/openshift/kubernetes.git
-    cd kubernetes
+```
+oc adm policy remove-scc-from-group anyuid system:authenticated system:serviceaccounts
+oc adm policy remove-scc-from-group privileged system:authenticated system:serviceaccounts
+```
 
-4. Place the \`oc\` binary for that cluster in your PATH
-5. Run the conformance test:
+This conformance report is generated by the OpenShift CI infrastructure. The canonical source location for this test script is located at https://github.com/openshift/kubernetes/blob/release-4.8/openshift-hack/conformance-k8s.sh
 
-    openshift-hack/conformance-k8s.sh
+This file was generated by:
+
+  Commit $( git rev-parse HEAD || "<commit>" )
+  Tag    ${version}
+
+Periodic conformance tests are run against release branches and reported at https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-kubernetes-release-4.8-k8s-conformance
 
-Nightly conformance tests are run against release branches and reported https://openshift-gce-devel.appspot.com/builds/origin-ci-test/logs/periodic-ci-origin-conformance-k8s/
 END
 
-version="$(grep k8s.io/kubernetes go.sum | awk '{print $2}' | sed s+/go.mod++ )"
 os::log::info "Running Kubernetes conformance suite for ${version}"
 
-# Execute OpenShift prerequisites
 # Disable container security
 oc adm policy add-scc-to-group privileged system:authenticated system:serviceaccounts
 oc adm policy add-scc-to-group anyuid system:authenticated system:serviceaccounts
-# Mark the master nodes as unschedulable so tests ignore them
-oc get nodes -o name -l 'node-role.kubernetes.io/master' | xargs -L1 oc adm cordon
-unschedulable="$( ( oc get nodes -o name -l 'node-role.kubernetes.io/master'; ) | wc -l )"
-# TODO: undo these operations
-
-# Execute Kubernetes prerequisites
-make WHAT=cmd/kubectl
-make WHAT=test/e2e/e2e.test
-make WHAT=vendor/github.com/onsi/ginkgo/ginkgo
-PATH="${OS_ROOT}/_output/local/bin/$( os::build::host_platform ):${PATH}"
-export PATH
 
 kubectl version  > "${test_report_dir}/version.txt"
 echo "-----"    >> "${test_report_dir}/version.txt"
 oc version      >> "${test_report_dir}/version.txt"
 
-# Run the test, serial tests first, then parallel
+# Run the tests
+
+wget https://github.com/vmware-tanzu/sonobuoy/releases/download/v0.55.1/sonobuoy_0.55.1_linux_amd64.tar.gz
+tar xf sonobuoy_0.55.1_linux_amd64.tar.gz
```

Comment:
sonobuoy versions are not aligned with kubernetes versions, however, e2e/conformance tests are.

sonobuoy may not run the correct tests for the cluster version, unless they have fixed it recently, something I don't know honestly